civis/cli/_cli_commands.py

```diff
@@ -85,6 +86,71 @@ def notebooks_download_cmd(notebook_id, path):
             f.write(lines)
 
 
+@click.command('new')
+@click.argument('language', type=click.Choice(['python3', 'python2', 'r']),
+                default='python3')
+@click.option('--mem', type=int, default=None,
+              help='Memory allocated for this notebook in MiB.')
+@click.option('--cpu', type=int, default=None,
+              help='CPU available for this notebook in 1/1024 of a core.')
+def notebooks_launch_cmd(language='python3', mem=None, cpu=None):
+    """Create a new notebook and open it in the browser."""
+    client = civis.APIClient()
+    kwargs = {'memory': mem, 'cpu': cpu}
+    kwargs = {k: v for k, v in kwargs.items() if v is not None}
+    new_nb = client.notebooks.post(language=language, **kwargs)
+    print("Created new {} notebook with ID {} .".format(language, new_nb.id))
```

Comment:
You might consider using `click.echo` here, BTW. http://click.pocoo.org/6/quickstart/#echoing

They claim it is better between python 2 and 3. I am not sure if we care.